data/tlg0627/tlg001/tlg0627.tlg001.1st1K-grc1.xml

```diff
@@ -67,7 +66,7 @@
                             <date>1839</date>
                         </imprint>
                     </monogr>
```

Comment:
I see this is volume 1, so `<biblScope unit="volume">1</biblScope>` needs to be indicated here